src/components/draw/feature-sub-content.vue

```diff
@@ -61,38 +64,60 @@ function onClickSearch() {
     <!-- Spacer -->
     <div class="min-h-3"></div>
 
-    <FeatureMeasurements :feature="feature" />
+    <FeatureMeasurements
+      :feature="feature"
+      :isEditingFeature="isEditingFeature"
+    />
 
     <!-- Default display feature options -->
     <div class="flex justify-between items-center mt-1">
       <div>
         <!-- Button edit feature: EDIT/END EDITION -->
         <button
+          data-cy="featItemToggleEdit"
           class="lux-btn-primary"
           @click="emit('toggleEditFeature', props.feature)"
         >
           {{ isEditingFeature ? t('Terminer édition') : t("Editer l'objet") }}
         </button>
       </div>
 
-      <div>
+      <div data-cy="featItemActions">
```

Comment:
appearence is different from v3 with long translated labels, better in my opinion since the action buttons disappeared in V3:
v3: 
![image](https://github.com/user-attachments/assets/0c8ebc4e-1e6c-43a4-a8dd-c9203949888e)

v4: 
![image](https://github.com/user-attachments/assets/4a5cc9fa-e1d3-4b73-9414-99eba577ba9c)

the layout in v4 seems better, but still a bit chunky, maybe just add some left margin to the action button group ?

Reply:
![image](https://github.com/user-attachments/assets/47d664f0-4c89-46d1-a45b-e2926c070317)

fixed! thx for reporting this